.github/workflows/release.yml

```
@@ -0,0 +1,35 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - main
+
```

[ask] According to the [`changesets/action` README](https://github.com/changesets/action#without-publishing), `concurrency` is set in the example. Do we also need it?

```suggestion

concurrency: ${{ github.workflow }}-${{ github.ref }}

```

---

Perhaps we should align with the official document, I've never used this previously.